spark/src/main/java/org/opensearch/sql/spark/data/constants/SparkConstants.java

```diff
@@ -8,13 +8,22 @@
 public class SparkConstants {
   public static final String EMR = "emr";
   public static final String STEP_ID_FIELD = "stepId.keyword";
-  // TODO should be replaced with mvn jar.
+
+  public static final String JOB_ID_FIELD = "jobRunId";
+
+  public static final String STATUS_FIELD = "status";
+
+  public static final String DATA_FIELD = "data";
+
+  public static final String ERROR_FIELD = "error";
+
+  // EMR-S will download JAR to local maven
   public static final String SPARK_SQL_APPLICATION_JAR =
-      "s3://flint-data-dp-eu-west-1-beta/code/flint/sql-job.jar";
+      "file:///home/hadoop/.ivy2/jars/org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar";
   public static final String SPARK_RESPONSE_BUFFER_INDEX_NAME = ".query_execution_result";
   // TODO should be replaced with mvn jar.
   public static final String FLINT_INTEGRATION_JAR =
-      "s3://spark-datasource/flint-spark-integration-assembly-0.1.0-SNAPSHOT.jar";
+      "s3://flint-data-dp-eu-west-1-beta/code/flint/sql-job.jar";
```

Comment:
revert this, it is for Spark datasource, not used in flint.

Reply:
reverted